src/BTCPayServer.Lightning.CLightning/ShortChannelId.cs

```diff
@@ -0,0 +1,29 @@
+using System.Globalization;
+using System.Linq;
+
+namespace BTCPayServer.Lightning.CLightning
+{
+    // clightning-specific representation of channel id
+    public class ShortChannelId
+    {
+        public ShortChannelId (string data)
```

Comment:
Should be `static ShortChannelId Parse(string str)` the only constructor which exist should initialized BlockHeight, BlockIndex and TxOutIndex based on the parameters.